src/components/implementation/tests/bench_dpdk/doc.md

```diff
@@ -0,0 +1,53 @@
+## How to perform this DPDK test?
```

Comment:
Given the scripts and config files in this directory, don't these commands need to be run in this directory? If so, say so.